Workarounds/MarioKart8_PortNumber/patch_port.asm

```diff
@@ -0,0 +1,7 @@
+[MarioKart8_v81]
```

Comment:
```suggestion
[MarioKart8_ForceMultiplayerPortNumber_v81]
```
This would make it easier to identify in the log.txt (when graphic pack logging is enabled under Debug->Logging) and PPC debugger.